ruby/extractor/Cargo.toml

```diff
@@ -27,14 +28,8 @@ encoding = "0.2"
 lazy_static = "1.4.0"
 # Ideally, we'd like to pull this in via a relative path.
 # However, our bazel/rust tooling chokes on this, c.f. https://github.com/bazelbuild/rules_rust/issues/1525
-# Therefore, to break that dependency, we depend on it via a git dependency instead.
-# We should change this back to a path dependency once this issue is fixed.
-# We can't depend on this without a rev/branch specification, as the rules_rust code assumes the default branch
-# is called `master`, and if we pull this in with `branch=main`, then `cargo` works (and pins this at th current git SHA
-# of lock-file update time, but `rules_rust` pins generates a bazel rule that unconditionally downloads `main`, which
-# breaks build hermeticity. So, rev-pinning it is.
-# See also https://github.com/bazelbuild/rules_rust/issues/2502.
-codeql-extractor = { git = "https://github.com/github/codeql.git", rev = "0dbce3d077f6f31a8d660aea104ee31cacf6bacd" }
+# Therefore, we have a pretty bad hack in place instead, see README.md in the codeql-extractor-fake-crate directory.
+codeql-extractor = { path = "codeql-extractor-fake-crate" }
 
 [patch.crates-io]
-tree-sitter = {git = "https://github.com/redsun82/tree-sitter.git", rev = "1f5c1112ceaa8fc6aff61d1852690407670d2a96"}
+tree-sitter = { git = "https://github.com/redsun82/tree-sitter.git", rev = "1f5c1112ceaa8fc6aff61d1852690407670d2a96" }
```

Comment:
Why do we have our own fork of tree-sitter?

Reply:
That's unrelated to this PR, and is I believe no longer necessary:
https://github.com/tree-sitter/tree-sitter/pull/3352 was merged (which broke our build otherwise), and we're now waiting on tree-sitter cutting a new release for us to pull in that fix via crates.io instead of git.